Bio/SeqIO/AbiIO.py

```diff
@@ -203,7 +422,11 @@ def _abi_parse_header(header, handle):
         # only parse desired dirs
         key = _bytes_to_string(dir_entry[0])
         key += str(dir_entry[1])
-        if key in (list(_EXTRACT) + _SPCTAGS):
+
+        global_tag_listing = []
+        for x in _INSTRUMENT_SPECIFIC_TAGS:
+            global_tag_listing += _INSTRUMENT_SPECIFIC_TAGS[x].keys()
+        if key in global_tag_listing:
```

Comment:
Another comment :) ~

I see that this is done inside the `while` loop (i.e. the `global_tag_listing` list being repeatedly rebuilt). Would be an idea to create if before the `while` loop?

Also, you can shorten the code a bit by working on the dictionary `values()` directly. Something like:

```
global_tag_listing = []
for t in _INSTRUMENT_SPECIFIC_TAGS.values():
    global_tag_listing += t.keys()
```